sql_test.go

```diff
@@ -42,12 +45,94 @@ func TestMigrations(t *testing.T) {
 	}
 }
 
+func TestSQLiteConcurreny(t *testing.T) {
+	ctx, s := setupDB(t)
+
+	var wg sync.WaitGroup
+
+	var errs []error
+
+	otLoc := otLocation{
+		Accuracy:      iptr(5), // TODO - should this allow nulls in the DB? OT has it as optional
+		TimestampUnix: int(time.Now().Unix()),
+	}
+	otLocb, err := json.Marshal(&otLoc)
+	if err != nil {
+		t.Fatal(err)
+	}
+	otMsg := owntracksMessage{
+		Type: "location",
+		Data: otLocb,
+	}
+
+	for i := 0; i < 10; i++ {
+		wg.Add(3)
+
+		go func() {
+			defer wg.Done()
+
+			for i := 0; i < 10; i++ {
+				if err := s.AddOTLocation(ctx, otMsg); err != nil {
+					errs = append(errs, err)
```

Comment:
I'm pretty sure it's not safe to `append` in multiple goroutines for the same slice. I usually use a `chan error` for this kind of thing.